modules/apps/layout/layout-taglib/src/main/java/com/liferay/layout/taglib/internal/servlet/ServletContextUtil.java

```diff
@@ -106,8 +106,8 @@ public static InfoListRendererTracker getInfoListRendererTracker() {
 		return _layoutDisplayPageProviderTracker;
 	}
 
-	public static LayoutListRetrieverTracker getLayoutListRetrieverTracker() {
-		return _layoutListRetrieverTracker;
+	public static LayoutListRetrieverRegistry getLayoutListRetrieverTracker() {
```

Comment:
@W2jx Please also rename methods that returns instance of the classes you renamed.